### PR TITLE
AudioIO: add OpenCV audio reading to dnn speech recognition sample

### PR DESCRIPTION
**Pull request by SinM9**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [ ] I agree to contribute to the project under Apache 2 License.
- [ ] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [ ] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

```
buildworker:Custom Win=windows-3
build_image:Custom Win=msvs2019-opengl
```

### CODE REVIEW

File: samples/dnn/speech_recognition.py [hunk 1]
```diff
@@ -412,6 +406,63 @@ def predict(features, net, decoder):
     prediction = decoder.decode(output.squeeze(0))
     return prediction[0]
 
+def readAudioFile(file, audioStream):
+    cap = cv.VideoCapture(file)
+    samplingRate = 16000
+    params = np.asarray([cv.CAP_PROP_AUDIO_STREAM, audioStream,
```
Comment (alalek) [inline samples/dnn/speech_recognition.py:412]: > np.asarray

please use tuple / list.

**Review by alalek — COMMENTED** [review]
Thank you!

**Review by MaximMilashchenko — APPROVED** [review]

**Comment by alalek** [comment]
:+1:

**Pull request merged**